detail.py

```diff
@@ -0,0 +1,31 @@
+from django_mako_plus import view_function, jscontext
+from django.http import HttpResponseRedirect
+from catalog import models as cmod
+import math
+
+def fix_html(x):
+    x = x.replace("&","&amp")
+    x = x.replace("<","&lt")
+    x = x.replace(">","&gt")
+    x = x.replace("\"","&quot")
+    x = x.replace("\'","&#39")
+    return x
```

Comment:
I think there is a better way to accomplish what you are doing here with a { | h} on the actual html page instead of having a completely separate function

Reply:
I didn't know that! Thanks! I will look into that.